ci-operator/step-registry/openshift-qe/crd-scale/openshift-qe-crd-scale-commands.sh

```
@@ -51,7 +51,7 @@ fi
 REPO_URL="https://github.com/cloud-bulldozer/e2e-benchmarking";
 LATEST_TAG=$(curl -s "https://api.github.com/repos/cloud-bulldozer/e2e-benchmarking/releases/latest" | jq -r '.tag_name');
 TAG_OPTION="--branch $(if [ "$E2E_VERSION" == "default" ]; then echo "$LATEST_TAG"; else echo "$E2E_VERSION"; fi)";
```

Shellcheck (0.11.0)

[warning] TAG_OPTION appears unused. Verify use (or export if used externally).

(SC2034)